<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAirlineIcaoForm</class>
 <widget class="QFrame" name="CAirlineIcaoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Airline ICAO</string>
  </property>
  <layout class="QVBoxLayout" name="vl_AirlineIcaoCode">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_AirlineIcao">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Airline ICAO</string>
     </property>
     <layout class="QGridLayout" name="gl_AirlineIcao" columnstretch="0,1,0">
      <item row="5" column="1">
       <widget class="QWidget" name="wi_Timestamps" native="true">
        <layout class="QHBoxLayout" name="hl_Timestamps">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="le_Updated">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="BlackGui::Components::CDbCountrySelectorComponent" name="country_Selector">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="wi_IdAndIcon" native="true">
        <layout class="QHBoxLayout" name="hl_IdAndIcon">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="le_Id">
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Id</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tb_Paste">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../../misc/misc.qrc">
             <normaloff>:/diagona/icons/diagona/icons/clipboard-paste.png</normaloff>:/diagona/icons/diagona/icons/clipboard-paste.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_AirlineIcon">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="BlackGui::CLabelAndIcon" name="lai_Id">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="wi_DesignatorVa" native="true">
        <layout class="QHBoxLayout" name="hl_DesignatorRank">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="BlackGui::Components::CDbAirlineIcaoSelectorComponent" name="selector_AirlineDesignator">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_Va">
           <property name="text">
            <string>VA?</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb_Military">
           <property name="text">
            <string>Mil.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbl_Wtc">
        <property name="text">
         <string>Country:</string>
        </property>
        <property name="buddy">
         <cstring>country_Selector</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_IcaoDesignator">
        <property name="text">
         <string>Designator/VA?:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_Name">
        <property name="text">
         <string>Name:</string>
        </property>
        <property name="buddy">
         <cstring>selector_AirlineName</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbl_Timestamp">
        <property name="text">
         <string>Last updated:</string>
        </property>
        <property name="buddy">
         <cstring>le_Updated</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="le_TelephonyDesignator">
        <property name="placeholderText">
         <string>How airline is called via radio</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_TelephonyDesignator">
        <property name="text">
         <string>Telephony:</string>
        </property>
        <property name="buddy">
         <cstring>le_TelephonyDesignator</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="BlackGui::CDropSite" name="drop_DropData">
        <property name="text">
         <string>Drop data here</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" rowspan="7">
       <widget class="BlackGui::Editors::CValidationIndicator" name="val_Indicator">
        <property name="minimumSize">
         <size>
          <width>16</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="BlackGui::Components::CDbAirlineNameSelectorComponent" name="selector_AirlineName">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::CLabelAndIcon</class>
   <extends>QFrame</extends>
   <header>blackgui/labelandicon.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Editors::CValidationIndicator</class>
   <extends>QFrame</extends>
   <header>blackgui/editors/validationindicator.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::CDropSite</class>
   <extends>QLabel</extends>
   <header>blackgui/dropsite.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAirlineIcaoSelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbairlineicaoselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbAirlineNameSelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbairlinenameselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Components::CDbCountrySelectorComponent</class>
   <extends>QFrame</extends>
   <header>blackgui/components/dbcountryselectorcomponent.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>le_Id</tabstop>
  <tabstop>cb_Va</tabstop>
  <tabstop>cb_Military</tabstop>
  <tabstop>le_TelephonyDesignator</tabstop>
  <tabstop>selector_AirlineName</tabstop>
  <tabstop>country_Selector</tabstop>
  <tabstop>le_Updated</tabstop>
 </tabstops>
 <resources>
  <include location="../../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
